nautobot/docs/user-guide/administration/upgrading/from-v2/tables/v3-code-removals.yaml

```diff
@@ -1,57 +1,55 @@
+  Replacement: "nautobot.dcim.api.serializers.CableTerminationModelSerializerMixin"
+- Removed: "nautobot.dcim.api.serializers.ConnectedEndpointSerializer"
+  Replacement: "nautobot.dcim.api.serializers.PathEndpointModelSerializerMixin"
+- Removed: "nautobot.extras.filters.CreatedUpdatedFilterSet"
+  Replacement: "nautobot.extras.filters.mixins.CreatedUpdatedModelFilterSetMixin"
+- Removed: "nautobot.extras.filters.RelationshipModelFilterSet"
+  Replacement: "nautobot.extras.filters.mixins.RelationshipModelFilterSetMixin"
+- Removed: "nautobot.extras.filters.CustomFieldModelFilterSet"
+  Replacement: "nautobot.extras.filters.mixins.CustomFieldModelFilterSetMixin"
+- Removed: "nautobot.extras.filters.LocalContextFilterSet"
+  Replacement: "nautobot.extras.filters.mixins.LocalContextModelFilterSetMixin"
+- Removed: "nautobot.extras.forms.forms.CustomFieldBulkCreateForm"
+  Replacement: "nautobot.extras.forms.mixins.CustomFieldBulkCreateForm"
+- Removed: "nautobot.extras.forms.mixins.AddRemoveTagsForm"
+  Replacement: "nautobot.extras.forms.mixins.TagsBulkEditFormMixin"
+- Removed: "nautobot.extras.forms.mixins.CustomFieldBulkEditForm"
+  Replacement: "nautobot.extras.forms.mixins.CustomFieldModelBulkEditFormMixin"
+- Removed: "nautobot.extras.forms.mixins.CustomFieldModelForm"
+  Replacement: "nautobot.extras.forms.mixins.CustomFieldModelFormMixin"
+- Removed: "nautobot.extras.forms.mixins.RelationshipModelForm"
+  Replacement: "nautobot.extras.forms.mixins.RelationshipModelFormMixin"
+- Removed: "nautobot.extras.forms.mixins.StatusBulkEditFormMixin"
+  Replacement: "nautobot.extras.forms.mixins.StatusModelBulkEditFormMixin"
+- Removed: "nautobot.extras.forms.mixins.StatusFilterFormMixin"
+  Replacement: "nautobot.extras.forms.mixins.StatusModelFilterFormMixin"
+- Removed: "nautobot.tenancy.filters.TenancyFilterSet"
+  Replacement: "nautobot.tenancy.filters.mixins.TenancyModelFilterSetMixin"
+- Removed: "nautobot.core.testing.filters.FilterTestCases.NameOnlyFilterTestCase"
+  Replacement: "nautobot.core.testing.filters.FilterTestCases.FilterTestCase"
+- Removed: "nautobot.core.testing.filters.FilterTestCases.NameSlugFilterTestCase"
+  Replacement: "nautobot.core.testing.filters.FilterTestCases.FilterTestCase"
+- Removed: "nautobot.extras.choices.CustomLinkButtonClassChoices"
+  Replacement: "nautobot.extras.choices.ButtonClassChoices"
+- Removed: "nautobot.extras.plugins.PluginConfig"
+  Replacement: "nautobot.extras.plugins.NautobotAppConfig"
+- Removed: "nautobot.extras.plugins.PluginTemplateExtension"
+  Replacement: "nautobot.extras.plugins.TemplateExtension"
+- Removed: "nautobot.extras.plugins.PluginBanner"
+  Replacement: "nautobot.extras.plugins.Banner"
+- Removed: "nautobot.extras.plugins.PluginFilterExtension"
+  Replacement: "nautobot.extras.plugins.FilterExtension"
+- Removed: "nautobot.extras.plugins.PluginCustomValidator"
+  Replacement: "nautobot.extras.plugins.CustomValidator"
```

Comment:
This is not related to this PR, but was just updating the table to match the one in pylint-nautobot.